code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -172,7 +172,9 @@
 	var/list/roundstart_wizards = list()
 
 /datum/dynamic_ruleset/roundstart/wizard/acceptable(population=0, threat=0)
-	if(GLOB.wizardstart.len == 0)
+	if(!check_candidates())
+		return FALSE
```

Comment:
this is also already handled by `ready()`. acceptable should not fail for invalid candidates